### PR TITLE
added openai-rev package

### PR DESCRIPTION
**Pull request by sudouser777**
1. added wrapper module for using all API's
2. moved all sub-packages(quora, you and theb etc..) inside openai_rev
3. added pyproject.toml

### CODE REVIEW

**Comment by sudouser777** [comment]
@xtekky I have tested the changes by installing the package locally, you can test once before merging
once you merge the changes, we can add github action to publish the package

File: openai_rev/openai_rev.py [hunk 1]
```diff
@@ -32,3 +36,7 @@ def __poe_service(cls, prompt: str, **kwargs) -> str:
     @classmethod
     def __fore_front_service(cls, prompt: str, **kwargs) -> str:
         return forefront.Completion.create(prompt=prompt, **kwargs).text
+
+    @classmethod
+    def __theb_service(cls, prompt: str, **kwargs):
+        return ''.join(theb.Completion.create(prompt=prompt))
```
Comment (AymaneHrouch) [inline openai_rev/openai_rev.py:42]: You're not using `cls`, can you make it staticmethod instead of classmethod?

File: openai_rev/openai_rev.py [hunk 2]
```diff
@@ -32,3 +36,7 @@ def __poe_service(cls, prompt: str, **kwargs) -> str:
     @classmethod
     def __fore_front_service(cls, prompt: str, **kwargs) -> str:
```
Comment (AymaneHrouch) [inline openai_rev/openai_rev.py:37]: Same here

File: openai_rev/theb/__init__.py [hunk 3]
```diff
@@ -14,20 +16,20 @@ class Completion:
     message_queue = Queue()
     stream_completed = False
 
-    def request(prompt: str):
+    @classmethod
+    def request(cls, prompt: str):
```
Comment (AymaneHrouch) [inline openai_rev/theb/__init__.py:20]: Same.

**Review by AymaneHrouch — CHANGES_REQUESTED** [review]
Can you add `pydantic` to requirements please?

**Comment by xtekky** [comment]
hi will look at this tmrw

**Comment by sudouser777** [comment]
> Can you add `pydantic` to requirements please?

@AymaneHrouch pydantic is already there in requirements

File: openai_rev/theb/__init__.py (hunk 3 above)
Comment (sudouser777) [reply openai_rev/theb/__init__.py]: updated

**Review by AymaneHrouch — APPROVED** [review]

**Comment by xtekky** [comment]
Hi, idk if this is a lot asked, but could you please update files and make a nice documentation, will work on it tmrw, and name package `gpt4free`.

**Comment by sudouser777** [comment]
@xtekky I'm closing the pull request, a lot of conflicts and duplicated files because of packaging changes

**Pull request closed without merging**